### PR TITLE
Update so that the appliction/json Content-Type is set

### PR DESCRIPTION
**Pull request by sparks**
This will let it work properly as a slash command also. That way all our single-channel users can check their debt status

Although there is a 3sec timeout on slack slash commands which we're close too

### CODE REVIEW

File: slask.py [hunk 1]
```diff
@@ -17,15 +17,15 @@
 import debt_utils
 from config import config
 
-def handle_message(message):
+def handle_message(command):
     # 'help'
```
Comment (cmyr) [inline slask.py:21]: I mean, it isn't a _command_ until we've parsed it and figured out if it's telling us anything? Also if I was going to rename the param I'd also rename the function. 😛

**Comment by cmyr** [comment]
I don't know what the api for the `/` commands is, and it isn't totally clear what this changes. Not that I don't believe you. Happy to merge if you think it works?

File: slask.py (hunk 1 above)
Comment (sparks) [inline slask.py:21]: Sure I mean mainly I thought I needed to pass in text from another format so I was going the `get` outside the function.

**Comment by sparks** [comment]
It will work as is as soon as you push it (it works now except the JSON isn't parsed because there's no header)

**Pull request merged**